client/my-sites/plugins/plugin-meta/index.jsx

```diff
@@ -150,18 +166,21 @@ export default React.createClass( {
 	},
 
 	getUpdateWarning() {
-		const newVersions = this.getAvailableNewVersions();
+		const newVersions = getAvailableNewVersions( this.props );
 		if ( newVersions.length > 0 ) {
 			if ( this.props.selectedSite ) {
+				const textVersion = this.translate( 'Version' );
+				const textIsAvailable = this.translate( 'is available' );
+
 				return (
 					<Notice
 						status="is-warning"
 						className="plugin-meta__version-notice"
 						showDismiss={ false }
 						icon="sync"
-						text={ i18n.translate( 'A new version is available.' ) }>
+						text={ `${ textVersion } ${ newVersions[ 0 ].newVersion } ${ textIsAvailable }` }>
```

Comment:
This is how it's done in https://github.com/Automattic/wp-calypso/pull/8124/commits/4a9690df69f0ab21204f6896e887ca6e24a7a498 :-) Not sure why you've seen an older version. Let me know if something is awry with the PR.